x-pack/plugins/triggers_actions_ui/public/application/sections/common/components/alert_quick_edit_buttons.tsx

```diff
@@ -147,72 +148,74 @@ export const AlertQuickEditButtons: React.FunctionComponent<ComponentOpts> = ({
   }
 
   return (
-    <Fragment>
+    <EuiContextMenuPanel>
```

Comment:
Not sure if this is new or caused by this PR but the first EuiContextMenuItem shows as highlighted for me (blue background).

<img width="207" alt="Screen Shot 2020-04-15 at 11 05 50 AM" src="https://user-images.githubusercontent.com/3694571/79355602-e1ed1400-7f0b-11ea-824a-0df0330e2eef.png">

Reply:
Set `hasFocus` prop to false in `EuiContextMenuPanel`